manifests/app/dreamkast/overlays/production/main/hpa.yaml

```
@@ -33,8 +33,8 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: dreamkast-ui
-  minReplicas: 20
-  maxReplicas: 30
+  minReplicas: 4
```

```suggestion
  minReplicas: 1
```

UIに関してはイベント開催してないときはトラフィックゼロなので、1インスタンスでOKです！